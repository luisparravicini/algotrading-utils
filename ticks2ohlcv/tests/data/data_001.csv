1450000005,120.00000,0.00000000
1450000062,120.00000,0.00000000
1450000138,120.00000,0.00000000
