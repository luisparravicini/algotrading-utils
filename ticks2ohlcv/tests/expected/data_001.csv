date,price,volume
1450000005,120.0,0.4
1450000062,120.0,0.4
1450000138,120.0,0.4
